tools/editor/project_export.cpp

```diff
@@ -602,6 +607,68 @@ void ProjectExportDialog::custom_action(const String&) {
 
 }
 
+bool ProjectExportDialog::_check_android_setting(const Ref<EditorExportPlatform>& exporter) {
+
+	bool is_debugging = exporter->get("debug/debugging_enabled");
+	String release = exporter->get("keystore/release");
+	String user = exporter->get("keystore/release_user");
+	String password = exporter->get("keystore/release_password");
+
+	if (!is_debugging && (release=="" || user=="" || password=="")){
+		// make & show custom alert message for Android release keystore infomation
+		AcceptDialog* error_release = memnew(AcceptDialog);
+		VBoxContainer* vb = memnew(VBoxContainer);
+		vb->set_margin(MARGIN_LEFT, 10*EDSCALE);
+		vb->set_margin(MARGIN_TOP, 10*EDSCALE);
+		Label* msg = memnew(Label);
+		msg->set_text(TTR("Fill \"Keystore\" section to export Android release apk.\n\nHow to make a keystore :"));
+		LineEdit* cmd = memnew(LineEdit);
+		cmd->set_expand_to_text_length(true);
+		cmd->set_editable(false);
+		cmd->set_text("keytool -genkey -v -keystore filename.keystore -alias alias_name -keyalg RSA -keysize 2048 -validity 10000 ");
+		// Ctrl+shift+v does not working but Shift+ins does. don't know why.
+		Label* msg_paste = memnew(Label);
+		msg_paste->set_text(TTR("Copy it and paste on terminal with [Shift] + [Ins]."));
```

Comment:
Otherwise you can also leave it out and assume that people know how to paste stuff in their terminal. I think the only ones who could have trouble with it would be `cmd.exe` users, but they would hopefully have the idea to try a right-click for contextual menu :)